llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -382,6 +410,10 @@ void RISCVFrameLowering::determineFrameLayout(MachineFunction &MF) const {
   // Get the number of bytes to allocate from the FrameInfo.
   uint64_t FrameSize = MFI.getStackSize();
 
+  // QCI Interrupts use at least 96 bytes of stack space
+  if (RVFI->useQCIInterrupt(MF))
+    FrameSize = std::max<uint64_t>(FrameSize, QCIInterruptPushAmount);
```

Comment:
why is the `<uint64_t>` needed? I think both operands have the same type so it should be inferred

Reply:
I think a previous version of my patch had a different type for the Push Amount, but yes I will drop this.